favorit_js_data_type/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>My fovorite data type in JS</title>
+</head>
+<body bgcolor="#f6f8fa">
+	<h1  bordertype="solid">Arrays</h1>
+	<p>My favorite data type is an array! Because it can contain any types of data of any quantity, including arrays)</p>	
```

Comment:
Seems good, but the object also may contain any type of JS data types. Why the Array preferred for you ?

Reply:
In contrast ordinary objects, the contents of an array are ordered